cmd/cortex/main.go

```diff
@@ -110,8 +110,7 @@ func main() {
 	}
 
 	runtime.KeepAlive(ballast)
-	err = t.Stop()
-	util.CheckFatal("initializing cortex", err)
+	util.CheckFatal("stopping cortex", err)
```

Comment:
This is not checking any new error (rather the `err` from `t, err := cortex.New(cfg)`). If you intended to check for error from `t.Run()`, you need to have `err = t.Run()` outside of the `if`.

Reply:
You are correct. I've now removed this line completely. Everything now happens in Run anyway.